examples/web-wagmi/package.json

```diff
@@ -18,7 +18,7 @@
   "dependencies": {
     "@bundlr-network/client": "^0.11.9",
     "@ethersproject/providers": "^5.7.2",
-    "@lens-protocol/react-web": "1.3.1",
+    "@lens-protocol/react-web": "2.0.0-alpha.4",
```

Comment:
this is not needed, I just remove this file every time changeset generates it

Reply:
ok good to know, thanks